src/components/EditImageModal/index.jsx

```diff
@@ -543,9 +688,14 @@ export default class EditImageModal extends React.Component {
       buttonType="primary"
       onClick={this.onInsertImageButtonClick}
     />
-  )
+  );
 
   handleOpenModal = (event) => {
+    let isEventSourceEmpty = true;
+    if (event.detail.src) {
+      isEventSourceEmpty = false;
```

Comment:
I can't leave a comment on these lines, but I think you could use this variable for lines 709 and 713 as well.

Reply:
Cool, I'll check it out.